src/vendoring/tasks/vendor.py

```diff
@@ -60,7 +60,7 @@ def rewrite_file_imports(
 
     for lib in vendored_libs:
         text = re.sub(
-            rf"(\n\s*|^)import {lib}(\n\s*)",
+            rf"(\n\s*|^)import {lib}(\n\s*|\.)",
```

Comment:
This will not work. As described in issue #15 you can't successfully rewrite any import of type `import X.Y`. Your example:

    import package.main as main

would become 

    from _vendoring import package.main as main

which is invalid syntax.

My PR #24 allows the form `import mod as foo` though.

Reply:
Yes, I updated the patch, would you bother to have a look?

Reply:
looks like it should work. 

I'm not sure why it makes sense to make these vendoring regexes more complex though? Fixing this in upstream would be an easy one liner per import.